pisa-proxy/parser/mysql/src/grammar.y

```diff
@@ -5988,6 +5990,26 @@ show_create_table_stmt -> Box<ShowCreateTable>:
     }
 ;
 
+show_index_stmt -> Box<ShowIndexStmt>:
```

Comment:
I can't found `show_index_stmt` rule  in `https://github.com/mysql/mysql-server/blob/8.0/sql/sql_yacc.yy`, but i found `show_keys_stmt` , I think `show_keys_stmt`  should be used here.